smartsim/settings/slurmSettings.py

```diff
@@ -270,6 +274,17 @@ def set_walltime(self, walltime: str) -> None:
         """
         self.run_args["time"] = str(walltime)
 
+    def set_het_group(self, het_group: int) -> None:
```

Comment:
One other thought that I just had. Thrown an error here and also in `make_mpmd` in case someone tries to set a `het_group` and also `mpmd` at the same time.

Reply:
Yep!